Backend/Remora.Discord.API.Abstractions/API/Objects/Guilds/IGuild.cs

```diff
@@ -73,6 +73,11 @@ public interface IGuild
         /// </summary>
         Optional<IDiscordPermissionSet> Permissions { get; }
 
+        /// <summary>
+        /// Gets the voice region id for the guild.
+        /// </summary>
+        string Region { get; }
```

Comment:
This property is deprecated in the v9 API and was removed a few commits ago.

Reply:
I feel like I must have been looking at the wrong version of the docs (NO idea how) because not only do I clearly see that note now, I also see the property marked as both optional and nullable. 

Will revert.